code/style.css

```diff
@@ -1,4 +1,357 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+body {
+  background: linear-gradient(to bottom right, #F6EABE, 
+  #FDA65D, #FFA810);
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 400; 
+}
+
+header {
+  padding: 10px;
+  margin: -10px -8px 20px -8px;
+  border-radius: 2px;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  background: linear-gradient(
+      to right bottom,
+      rgba(255, 255, 255, 0.7),
+      rgba(255, 255, 255, 0.3)
+    ); 
+  box-shadow: 6px 6px 20px rgba(122,122,122,0.212);
+}
+
+header img {
+  margin: 5px;
+  grid-column-start: 2;
+  justify-self: center;
+  align-self: center;
+  filter: drop-shadow( 3px 3px 2px rgba(0, 0, 0, .5));
+}
+
+.meny img {
+  filter: drop-shadow( 3px 3px 2px rgba(0, 0, 0, .2));
+}
+
+nav {
+ display: flex;
+ font-family: 'Comfortaa', cursive;
+}
+
+nav a {
+  display: none;
+}
+
+nav a:hover {
+  background-color: #FDA65D;
+  border-radius: 50px;
+  box-shadow: 6px 6px 20px rgba(122,122,122,0.212);
+  animation-name: colorChange;
+  animation-duration: 5s; 
+}
+
+p {
+  line-height: 30px;
+  margin: 20px; 
+}
+
+a:visited, a:link {
+  color: black;
+  text-decoration: none;
+}
+
+/* BIG-NEWS */
+.bigNews {
+ padding: 5px;
+ margin: 0 -8px 0 -8px;
+  border-radius: 10px;
+  display: flex;
+  justify-content: space-evenly;
+  background: linear-gradient(
+  to right bottom,
+  rgba(255, 255, 255, 0.7),
+  rgba(255, 255, 255, 0.3)
+);
+  -webkit-backdrop-filter: blur(100px);
+  backdrop-filter: blur(10px);
+  box-shadow: 6px 6px 20px rgba(122,122,122,0.212);
+}
+
+.bigNews:hover {
+  animation-name: colorChange;
+  animation-duration: 5s;
+}
+
+.part1, .part2 {
+  padding: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.part1 img {
+ border-radius: 5px;
+ margin-left: 10px;
+}
+
+
+/* SMALL CARDS/ARTICLES */
+.cards {
+  margin: 40px 15px;
+  backdrop-filter: blur(10px);
+  display: flex;
+  justify-content: center;
+}
+
+.card1, .card2, .card3, .card4, .card5, .card6, .card7, .card8 {
```

Comment:
Hey! There is no need to create separate card classes like that. Next time try to create only one class called card and you can add the cart class in multiple divs :)